LaTeXTools.sublime-settings

```diff
@@ -282,6 +282,40 @@
 		]
 	},
 
+
+// ------------------------------------------------------------------
+// Additional fill all helper settings (for experts)
+// ------------------------------------------------------------------
+	// entries for the fill all helper
```

Comment:
It'd be good to have some documentation on the format of entries expected... Not everyone may be willing to read the Python code.